2023-02-02 02:09:29,840 INFO    MainThread:15964 [wandb_setup.py:_flush():75] Loading settings from C:\Users\bhata\.config\wandb\settings
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_setup.py:_flush():75] Loading settings from C:\aiornot\notebooks\wandb\settings
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_init.py:_log_setup():386] Logging user logs to C:\aiornot\notebooks\wandb\run-20230202_020929-2w5x91po\logs\debug.log
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_init.py:_log_setup():387] Logging internal logs to C:\aiornot\notebooks\wandb\run-20230202_020929-2w5x91po\logs\debug-internal.log
2023-02-02 02:09:29,841 INFO    MainThread:15964 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001DCB94225E0>
2023-02-02 02:09:29,842 INFO    MainThread:15964 [wandb_init.py:init():420] calling init triggers
2023-02-02 02:09:29,842 INFO    MainThread:15964 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2023-02-02 02:09:29,842 INFO    MainThread:15964 [wandb_init.py:init():471] starting backend
2023-02-02 02:09:29,842 INFO    MainThread:15964 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2023-02-02 02:09:29,846 INFO    MainThread:15964 [backend.py:ensure_launched():219] starting backend process...
2023-02-02 02:09:29,917 INFO    MainThread:15964 [backend.py:ensure_launched():224] started backend process with pid: 9504
2023-02-02 02:09:29,918 INFO    MainThread:15964 [wandb_init.py:init():480] backend started and connected
2023-02-02 02:09:29,931 INFO    MainThread:15964 [wandb_run.py:_label_probe_notebook():921] probe notebook
2023-02-02 02:09:29,932 INFO    MainThread:15964 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-02 02:09:29,932 INFO    MainThread:15964 [wandb_init.py:init():550] updated telemetry
2023-02-02 02:09:30,089 INFO    MainThread:15964 [wandb_init.py:init():581] communicating current version
2023-02-02 02:09:31,308 INFO    MainThread:15964 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.13.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-02-02 02:09:31,309 INFO    MainThread:15964 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2023-02-02 02:09:31,807 INFO    MainThread:15964 [wandb_init.py:init():624] starting run threads in backend
2023-02-02 02:09:36,824 INFO    MainThread:15964 [wandb_run.py:_console_start():1827] atexit reg
2023-02-02 02:09:36,825 INFO    MainThread:15964 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2023-02-02 02:09:36,825 INFO    MainThread:15964 [wandb_run.py:_redirect():1738] Wrapping output streams.
2023-02-02 02:09:36,826 INFO    MainThread:15964 [wandb_run.py:_redirect():1762] Redirects installed.
2023-02-02 02:09:36,827 INFO    MainThread:15964 [wandb_init.py:init():651] run started, returning control to user process
2023-02-02 02:09:47,146 INFO    MainThread:15964 [wandb_init.py:_pause_backend():299] pausing backend
2023-02-02 02:09:47,146 INFO    MainThread:15964 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-02-02 02:11:21,870 INFO    MainThread:15964 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2023-02-02 02:11:21,872 INFO    MainThread:15964 [wandb_run.py:_restore():1769] restore
2023-02-02 02:11:22,066 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 876
}

2023-02-02 02:11:22,429 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 876
}

2023-02-02 02:11:22,751 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10498
}

2023-02-02 02:11:22,858 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10498
}

2023-02-02 02:11:22,967 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10498
}

2023-02-02 02:11:23,078 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 876
  total_bytes: 10498
}

2023-02-02 02:11:23,185 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,297 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,406 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,515 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,624 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,732 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

2023-02-02 02:11:23,841 INFO    MainThread:15964 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10498
  total_bytes: 10498
}

